ui/src/app/pages/workspace/workspace-card.tsx

```diff
@@ -173,6 +182,13 @@ export const WorkspaceCard = fp.flow(withNavigation)(
       }
     }
 
+    workspaceCardNameTarget() {
+      const { tierAccessDisabled, workspace } = this.props;
+      return tierAccessDisabled
+        ? ' #'
+        : '/workspaces/' + workspace.namespace + '/' + workspace.id + '/data';
```

Comment:
nit: would be nicer with backticks, like `/workspaces/${namespace}` etc

Reply:
DOne